enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -466,7 +480,8 @@ impl<'a> Handler<'a> {
         }
 
         let cursor = self.block.cursor();
-        let untrusted = unsafe { cursor.alloc::<libc::timespec>(1).or(Err(libc::EMSGSIZE))? };
+        let (_cursor, untrusted) =
```

Comment:
```suggestion
        let (_, untrusted) =
```

Discard with naked underscore.